packages/code-du-travail-frontend/src/common/RelatedItems.js

```diff
@@ -1,165 +1,97 @@
-import {
-  getLabelBySource,
-  getRouteBySource,
-  SOURCES,
-} from "@socialgouv/cdtn-sources";
+import { getRouteBySource, SOURCES } from "@socialgouv/cdtn-sources";
 import {
   ArrowLink,
   Container,
   FlatList,
   Heading,
-  icons,
   theme,
 } from "@socialgouv/cdtn-ui";
 import Link from "next/link";
 import React from "react";
 import styled from "styled-components";
 
 import { matopush } from "../piwik";
-import { CallToActionTile } from "./tiles/CallToAction";
+
+const matoSelectRelated = (reco, selection) => {
+  matopush([
+    "trackEvent",
+    "selectRelated",
+    JSON.stringify({
+      reco,
+      selection,
+    }),
+  ]);
+};
 
 export const RelatedItems = ({ items = [] }) => {
-  const tileSources = [SOURCES.EXTERNALS, SOURCES.LETTERS, SOURCES.TOOLS];
+  const nonArticleSources = [SOURCES.EXTERNALS, SOURCES.LETTERS, SOURCES.TOOLS];
```

Comment:
is would use a function called "isArticle" here and then use !isArticle or or isArticle to avoid double negation but it' a minor imrpovement :)